core/Core/inputin.py

```diff
@@ -20,6 +20,11 @@ def inputin():
     try:
         global web
         web = raw_input(''+O+' [#] Target web address :> '+C)
+        if 'exit' in web:
+            print(R+' [-] Exiting...')
```

Comment:
```suggestion
            print(RED+' [-] Exiting...')
```
These need to be RED too  right? Just checking...